Code/Framework/AzCore/AzCore/std/any.h

```diff
@@ -83,6 +83,8 @@ namespace AZStd
             bool m_isPointer = false;
             // Should the object be stored on the heap? (never true for pointers)
             bool m_useHeap = false;
+            // The size of the value stored by this type
+            size_t m_valueSize;
```

Comment:
Can you initialize the m_valueSize member to 0, just to be safe

Reply:
Thanks, fixed!